server/handlers.go

```diff
@@ -518,6 +527,15 @@ func getCurrentApplication(r *http.Request) *models.Application {
 	return nil
 }
 
+func getTarget(a *models.Application, t string) (string, error) {
```

Comment:
I think you mean findTarget(a *models.Application, name string)(*models.Target, error) or?
else if find getTarget getting a target and returning a string strange.

I will change the name of this function and the return to *models.target later.